react/features/chat/components/native/ChatMessage.js

```diff
@@ -60,7 +63,7 @@ class ChatMessage extends AbstractChatMessage<Props> {
                                 && this._renderDisplayName()
                         }
                         <Linkify linkStyle = { styles.chatLink }>
-                            { messageText }
+                            { replaceAsciiEmoticon(messageText) }
```

Comment:
question: is this also applied for sent messages or just for received ones?

Reply:
no, this is only for rendering